src/deployment/android.md

```diff
@@ -260,7 +260,9 @@ Flutter to automatically depend on `androidx.multidex:multidex` and use a
 generated `FlutterMultiDexApplication` as the project's application.
 
 {{site.alert.note}}
-  Multidex support is natively included when targeting min sdk 21+.
+  Multidex support is natively included when targeting min sdk 21+. However,
+  it is not recommended to target API 21+ purely to resolve the multidex issue
+  as this may inadvertently exclude users running older devices.
```

Comment:
```suggestion
  Multidex support is natively included when targeting Android SDK 21 or later.
  However, it isn't recommended to target API 21+ purely to resolve the multidex issue
  as this might inadvertently exclude users running older devices.
```